bindings/mqtt/mqtt_integration_test.go

```diff
@@ -0,0 +1,112 @@
+package mqtt
+
+import (
+	"os"
+	"testing"
+	"time"
+
+	mqtt "github.com/eclipse/paho.mqtt.golang"
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dapr/components-contrib/bindings"
+	"github.com/dapr/kit/logger"
+)
+
```

Comment:
Please add a linter ignore statement so the golangci linter doesn't complain about the spelling of `mosquitto`